sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/ExpressionConverter.java

```diff
@@ -978,7 +978,7 @@ private RexNode convertResolvedStructFieldAccessInternal(RexNode referencedExpr,
     return rexBuilder().makeFieldAccess(referencedExpr, fieldIdx);
   }
 
-  private RexBuilder rexBuilder() {
+  public RexBuilder rexBuilder() {
```

Comment:
I am thinking this is not needed. In every Rel converter they can get RexBuilder.